### DISCUSSION TITLE
README.md Getting started

### DISCUSSION THREAD

**Discussion by mkastner** [zigzap/zap#43] [Q&A] [answered]
In the README.md it says:

```
With an existing zig project, adding zap to it is easy:

    Add zap to your build.zig.zon
    Add zap to your build.zig
```

Wouldn't that apply for projects created from scratch too?

**Comment by renerocksai** [comment, marked as answer]
Projects created from scratch are existing, too 😉

mtkastner ***@***.***> schrieb am Fr., 22. Sept. 2023, 09:09:
